x/upgrade/abci_test.go

```diff
@@ -1,26 +1,30 @@
 package upgrade_test
 
 import (
+	"reflect"
 	"testing"
 	"time"
 
+	"github.com/spf13/viper"
```

Comment:
tbh I don't see the need of depending on viper in any package other than `cli`. It can we mock it somehow?

Reply:
Agree here, I would tune the suite to use some other variable/option and not rely on Viper.

Reply:
Removed the viper usage and using set method in keeper instead.